src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -180,6 +180,29 @@ public sealed class DiagnosticLogManager : AgentService, IDiagnosticLogManager
                 }
             }
 
+            if (PlatformUtil.RunningOnLinux && !PlatformUtil.RunningOnRHEL6) {
```

Comment:
Please add debug message for the case when there is another platform, and this step was skipped

Reply:
Added message for else branch